### ISSUE TITLE
Please support shutters

### ISSUE THREAD

**Issue by rpdever** [arachnetech/homebridge-mqttthing#35]
Hello,

first of all thanks for this incredible Plugin! But it need the ability to integrate Shutters. Is there a possibility to integrate this feature the next weeks? 

I am using the awesome Devices of Shelly which are open source. Maybe this part of the API Reference will help you:

MQTT in Roller mode

When configured to operate in roller mode, MQTT topics used by Shelly Switch are:

shellies/shellyswitch-<deviceid>/roller/0 reports the current state: open, close while in motion, stop when not moving.
shellies/shellyswitch-<deviceid>/roller/0/command accepts open, close and stop.

If you are not able to integrate this the next weeks, please write me some instructions that I can do instead.

**Comment by rpdever** [comment]
Hi, 

maybe you will have time the next weeks again. Now the teased product above has some new MQTT possibilities:

MQTT in Roller mode

When configured to operate in roller mode, MQTT topics used by Shelly2 are:

shellies/shellyswitch-<deviceid>/roller/0 reports the current state: open, close while in motion, stop when not moving.
shellies/shellyswitch-<deviceid>/roller/0/command accepts rc (performs roller calibration), open, close and stop.
New in v1.4.0: roller mode position control. For this to work, the device must first be successfully calibrated, so that the time it takes for closing and opening are known.

shellies/shellyswitch-<deviceid>/roller/0/pos reports the current position in percent
shellies/shellyswitch-<deviceid>/roller/0/command/pos accepts a number between 0 and 100, which is target position in percent.


Have a nice holiday and a good new year!

**Comment by rpolenthon** [comment]
There is an updated and well working forked repository which supports different kinds of mqtt blinds, including the shelly 2's.

**Comment by ivanmoralesv** [comment]
> There is an updated and well working forked repository which supports different kinds of mqtt blinds, including the shelly 2's.

What fork? I haven’t found it :S

**Comment by rpdever** [comment]
Yes. Sorry i forgot to write the name of the account. Happy new Year!

**Comment by knstntn1** [comment]
Hi, as you are interested in my fork for shelly2 support:
Tim transferred the origin repository to me and i just committed my changes into the master. I‘ve deleted the fork, and updates will be made in the master repository. If you starred my fork for notifications, this will not work anymore. Also your forks should be synced with the master. Cheers, Konstantin

**Comment by rpolenthon** [comment]
Thanks for the information. The config example for the Shellys is really nice. But the last / in the main Topic is too much. You add it already with the code.

**Comment by arachnetech** [comment]
I've added support for blinds ("window coverings") in mqttthing too now - hopefully useful to someone.
